src/main/java/eu/miaplatform/service/ServiceOptions.java

```diff
@@ -0,0 +1,18 @@
+package eu.miaplatform.service;
+
+import lombok.*;
+
+@Data
+@AllArgsConstructor
+public class ServiceOptions extends InitServiceOptions {
+    private ReturnAs returnAs;
+    private int[] allowedStatusCodes;
+    private boolean isMiaHeaderInjected;
+
+    public ServiceOptions() {
```

Comment:
We should keep a similar interface to initserviceoptions with builder annotations

Reply:
Done